corehq/apps/accounting/models.py

```diff
@@ -1703,26 +1703,54 @@ def _send_ending_reminder_email(self, template, template_plaintext):
 
         email_html = render_to_string(template, context)
         email_plaintext = render_to_string(template_plaintext, context)
-        bcc = [settings.ACCOUNTS_EMAIL] if not self.is_trial else []
```

Comment:
I don't see this condition accounted for in the new code. Is this no longer necessary?

Reply:
That's right. Ops doesn't see value in being BCC'd on all of these, probably especially as our self-service user base grows.